lib/activerecord/delay_touching.rb

```diff
@@ -6,9 +6,9 @@ module DelayTouching
     extend ActiveSupport::Concern
 
     # Override ActiveRecord::Base#touch.
-    def touch(name = nil)
+    def touch(*names)
       if self.class.delay_touching? && !try(:no_touching?)
-        DelayTouching.add_record(self, name)
+        DelayTouching.add_record(self, names)
```

Comment:
No, it's entirely valid for names to be empty here. I can call `model.touch` with no arguments, which is valid and will end up touching `updated_at` rather than a specific column that could be specified in the names argument.